laser_slam/src/benchmarker.cpp

```diff
@@ -5,109 +5,219 @@
 #include <iomanip>
 #include <iostream>
 
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
No problem, boost is more compact, but it's definitely possible to do the same without!